lib/Xud.ts

```diff
@@ -141,11 +142,26 @@ class Xud extends EventEmitter {
       } else {
         this.logger.warn('RPC server is disabled.');
       }
+
+      this.bindXud();
     } catch (err) {
       this.logger.error(err);
     }
   }
 
+  private bindXud = () => {
```

Comment:
Why `bindXud`? Wouldn't `bindLnd(s)` be more appropriate?

Reply:
I was thinking to make it more generic for handling future events we need to listen to. But maybe I should just call this `bind`? That's the naming in Swaps.ts. Elsewhere we have "bind" functions they're binding a particular object, like `bindPeer` or `bindPool`.

Reply:
:+1: for `bind`